.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ lint-template: &lint-template
 jobs:
```

Comment:
You also need to update tox.ini. Check for other files too. Perhaps a grep for "3.11" across the project would help.

Reply:
make sense, gonna try it now.